consul/data_source_consul_acl_auth_method.go

```diff
@@ -90,27 +94,18 @@ func dataSourceConsulACLAuthMethodRead(d *schema.ResourceData, meta interface{})
 
 	authMethod, _, err := client.ACL().AuthMethodRead(name, qOpts)
 	if err != nil {
-		return fmt.Errorf("Failed to get auth-method: %v", err)
+		return fmt.Errorf("failed to get auth-method: %v", err)
```

Comment:
The error messqge was capitalized so when running Terraform the user facing message was 

```
│ Error: Failed to get auth-method: Get "http://localhost:8500/v1/acl/auth-method/not-found": dial tcp [::1]:8500: connect: connection refused
│ 
│   with data.consul_acl_auth_method.test,
│   on example.tf line 10, in data "consul_acl_auth_method" "test":
│   10: data "consul_acl_auth_method" "test" {
│ 
```

I think it is the standard in Go to not capitalize errors and it seems to me like no capitalization after semi-colons is more common that capitalization, though I'm not a native speaker and the style guides I've looked at are inconsistent.

Since the inconsistency in the error messages I return is something that have been slightly bugging me off, I finally settled on lowercase to start each error message to at least be consistent and started fixing the error messages in the files I've been updating.

In the end all error messages should all be consistent and be lowercase right after `Error: `:

```
│ Error: failed to get auth-method: Get "http://localhost:8500/v1/acl/auth-method/not-found": dial tcp [::1]:8500: connect: connection refused
│ 
│   with data.consul_acl_auth_method.test,
│   on example.tf line 10, in data "consul_acl_auth_method" "test":
│   10: data "consul_acl_auth_method" "test" {
│ 
```

Do you think I should revert that?

Reply:
I like the consistency and understand the reasoning. Happy to leave it that way. 
Creating a PR for the error messages alone would have made this PR much easier to review.